app/frontend/components/info_popup.js

```diff
@@ -209,6 +223,25 @@ class InfoPopup {
     $('#info-popup-content').empty();
     $('#info-popup-content').html(appInfo.innerHTML);
     $('#app-info-tabs').tabs({ heightStyle: "fill" });
+
+    if (this.platformHelper.isElectron()) {
+      $('#export-user-data-button').bind('click', async () => {
```

Comment:
Can we do it with vanilla JS, without jQuery?
```javascript
document.querySelector('#export-user-data-button').addEventListener('click', async () => {
  ...
});
```

Reply:
Ok!